components/validation/src/main/java/org/wso2/carbon/identity/x509Certificate/validation/X509CertificateValidationConstants.java

```diff
@@ -23,6 +23,9 @@
  */
 public class X509CertificateValidationConstants {
 
+    public static final String VALIDATOR_CONF_RESOURCE_PREFIX = "certificate/validator/";
```

Comment:
No usage of this.

Reply:
kept it because the deprecated methods have the usage

Reply:
sorry i saw it as already added one, not a newly added one. Thought existing code. My mistake.